docs/setup/tomcat.md

```diff
@@ -73,59 +78,96 @@ echo "" | openssl s_client -connect LDAPHOST:LDAPPORT -showcerts 2>/dev/null | o
 sudo keytool -import -alias cert_ldap -file /tmp/certfile.txt -keystore /etc/tomcat9/keystore
 ```
 
-### Finally, 
+### Finally,
+
 verify the list of keys in keystore:
 ```
 keytool -keystore /etc/tomcat9/keystore -list
 ```
 
+## geOrchestra datadir
+
+Since the 15.12 version, geOrchestra allows its main configuration settings to be done into a specific directory, the "georchestra Datadir".
+
+By convention, we will consider that the path `/etc/georchestra` will be dedicated to this datadir. We can bootstrap a default datadir by
```

Comment:
/can/have to/

Reply:
strictly speaking, the datadir could be elsewhere (that is what we do for our customers), I'd prefer to stay less restrictive than "have to", maybe a future ("we will bootstrap") would be more appropriate.